Tasks/VsTest/vstest.ts

```diff
@@ -130,6 +136,42 @@ function getTestAssemblies(): Set<string> {
     return new Set(testAssemblyFiles);
 }
 
+function addVstestDiagOption(argsArray: string[]) {
+    let sysDebug = tl.getVariable("System.Debug");
+    if (sysDebug !== undefined && sysDebug.toLowerCase() === "true") {
```

Comment:
All else conditions are printing error messages for different error conditions, so all of them are different.